Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG AWSCLI_VERSION="awscli-exe-linux-x86_64.zip"
 ENV AWSCLI_URL="https://awscli.amazonaws.com/${AWSCLI_VERSION}"
 
 # install tools needed for installation
-RUN yum install -y unzip git make gcc
+RUN dnf install -y unzip git make gcc
```

Comment:
What's the reasoning for changing from `yum` to `dnf` across the Dockerfile?

Reply:
We discussed offline that they are essentially the same, but I suggested making the change in a standalone PR.